flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/harness/TableAggregateHarnessTest.scala

```diff
@@ -161,7 +161,7 @@ class TableAggregateHarnessTest(mode: StateBackendMode) extends HarnessTestBase(
   private def createTableAggregateWithRetract()
       : (KeyedOneInputStreamOperatorTestHarness[RowData, RowData, RowData], Array[LogicalType]) = {
     val top3 = new Top3WithRetractInput
-    tEnv.registerFunction("top3", top3)
+    tEnv.createTemporarySystemFunction("top3", top3)
```

Comment:
here we can not use 
```scala
tEnv.createTemporarySystemFunction("top3", classOf[Top3WithMapView])
``` 
since `top3` is used in `flatAggregate` below